gems/pending/Gemfile

```diff
@@ -25,7 +25,7 @@ gem "fog-core",                "!=1.31.1",          :require => false
 gem "httpclient",              "~>2.5.3",           :require => false
 gem "image-inspector-client",  "~>1.0.0",           :require => false
 gem "kubeclient",              "=0.8.0",            :require => false
-gem "hawkular-client",         "~>0.1.2",           :require => false
+gem "hawkular-client",                    :require => false, :git => "git@github.com:hawkular/hawkular-client-ruby.git", :branch => "master"
```

Comment:
there is no released version to use not via git? (in any case, afaik if it's master branch, no need to specify it)